### ISSUE TITLE
 Cannot find module 'node-vantage'

### ISSUE THREAD

**Issue by bwired-nl** [Kamugo/node-vantage#1]
works like this:

``` js
var driver = require('./driver');
var driver = new driver();
driver.on('connect', function() {
  console.log('Connected to the Vantage');
  driver.on('loop', function(loop) {
    console.log(loop);
  });
});
```

**Comment by mpasa** [comment]
Hi @bwired-nl,

first of all, thank you for using my node vantage driver. I did it to fulfil my own needs, because I found all the other drivers very complicated. This means that I cannot guarantee any quality, and I don't know if I will improve it in the future. If you want to contribute I will be very grateful. The current version has been getting data from my Vantage Vue for almost a year without problems. I stream the data using socket.io to an express server, where it is shown in real time.

Talking about your problem, I've updated the documentation with the correct way to use it using the package in npm. I've also updated the serialport dependency, because the used in the package was having problems with Node 4.
